sylvia/src/into_response.rs

```diff
@@ -0,0 +1,90 @@
+use cosmwasm_std::{CosmosMsg, Empty, Response, StdError, StdResult, SubMsg};
+
+/// Trait converting `SubMsg` to one carrying another chain-custom message
+pub trait IntoMsg<C> {
+    fn into_msg(self) -> StdResult<SubMsg<C>>;
+}
+
+/// `SubMsg<Empty>` can be made into any `SubMsg<C>`
+impl<C> IntoMsg<C> for SubMsg<Empty> {
+    fn into_msg(self) -> StdResult<SubMsg<C>> {
+        let msg = match self.msg {
+            CosmosMsg::Wasm(wasm) => CosmosMsg::Wasm(wasm),
+            CosmosMsg::Bank(bank) => CosmosMsg::Bank(bank),
+            CosmosMsg::Staking(staking) => CosmosMsg::Staking(staking),
+            CosmosMsg::Distribution(distribution) => CosmosMsg::Distribution(distribution),
```

Comment:
As in prev comment - also enabled by the feature flag in `cosmwasm-std` (`staking` as well).